Cartfile.resolved

```diff
@@ -0,0 +1,2 @@
+github "sozorogami/JXHTTP" "2.0.3"
```

Comment:
Crap. Changed the Cartfile to not be pointing at my fork, but didn't `carthage update` after. I'll fix this.